autoware_control_msgs/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_control_msgs</name>
+  <version>1.0.0</version>
+  <description>Autoware control messages package.</description>
+
+  <maintainer email="mfc@leodrive.ai">M. Fatih Cırıt</maintainer>
+  <maintainer email="opensource@apex.ai">Apex.AI, Inc.</maintainer>
```

Comment:
Is `apex.ai` still involved ?

Reply:
@maxime-clem they are not, as fas as I know

Reply:
Yes, it's a good idea to write in the `NOTICE` file like `These messages are based on _auto messages that are mainly developed by Apex.AI`. :+1: